front/assets/style/base.scss

```diff
@@ -102,11 +107,20 @@
 
 /* dark theme styles */
 .theme--dark.v-btn {
-  background: $primary-color !important;
-  color: white !important;
+  background-color: $btn-default-background !important;
+  padding: 11px 24px !important;
+  color: $btn-default-color !important;
+  font-family: Avenir LT Std;
 
   &:hover {
-    background: $primary-color !important;
+    height: 46px !important;
```

Comment:
why  do we need to fix hover height?

Reply:
also it will not work properly even if we need different height because button could be different default height

Reply:
Yes. I am aware of button different sizes. 
I am not sure if it's a mistake but hover has a different height.
Check out the designs here - https://github.com/ezl/dailyidea.com/issues/482#issuecomment-658970929

@alisonuxdesign was this a mistake? does the button have a different height on hover

Reply:
Yes this a misktake @danleyb2, thank you for catching this. all buttons should be the same dimensions in all states (hover, disabled, regular)